crates/orchestrator/src/cli/mod.rs

```diff
@@ -403,8 +403,19 @@ pub mod validate_params {
         aws_event_bridge_args: &AWSEventBridgeCliArgs,
         aws_config_args: &AWSConfigCliArgs,
     ) -> Result<CronValidatedArgs, String> {
-        if aws_event_bridge_args.aws_event_bridge && aws_config_args.aws {
+        if (aws_event_bridge_args.aws_event_bridge_rule || aws_event_bridge_args.aws_event_bridge_schedule)
+            && aws_config_args.aws
+        {
+            let cron_type = if aws_event_bridge_args.aws_event_bridge_rule {
+                EventBridgeType::Rule
+            } else if aws_event_bridge_args.aws_event_bridge_schedule {
+                EventBridgeType::Schedule
+            } else {
+                panic!("No Cron service selected!")
+            };
```

Comment:
why not make a clap arg called event_bridge_type and make that an enum

Reply:
we would still need a boolean for event_bridge, just like we have to all other services cli args

we can change it to option<string> with additional boolean which makes sure that if the boolean is true all other args related to event bridge needs to be present